### ISSUE TITLE
db:rollback with view recreates the view

### ISSUE THREAD

**Issue by lowjoel** [SchemaPlus/schema_plus_views#7]

**Comment by ronen** [comment]
@lowjoel Thanks for the report -- confirmed that if you have a  `create_view` in a `change` migration:
```ruby
def change
    create_view :myview, "SELECT * FROM mytable"
end
```
then attempting a `db:rollback` will cause it to try to create the view again (and fail), instead of remove it.

Looking into it...

**Comment by lowjoel** [comment]
I vaguely remember that there needs to be a check on the migration environment before deciding whether to invert the transform. Thanks for checking.

**Comment by ronen** [comment]
Fixed, released 0.2.1
